store/tikv/kv.go

```diff
@@ -181,16 +184,17 @@ func newTikvStore(uuid string, pdClient pd.Client, spkv SafePointKV, client Clie
 		return nil, errors.Trace(err)
 	}
 	store := &tikvStore{
-		clusterID:   pdClient.GetClusterID(context.TODO()),
-		uuid:        uuid,
-		oracle:      o,
-		client:      client,
-		pdClient:    pdClient,
-		regionCache: NewRegionCache(pdClient),
-		kv:          spkv,
-		safePoint:   0,
-		spTime:      time.Now(),
-		closed:      make(chan struct{}),
+		clusterID:       pdClient.GetClusterID(context.TODO()),
+		uuid:            uuid,
+		oracle:          o,
+		client:          client,
+		pdClient:        pdClient,
+		regionCache:     NewRegionCache(pdClient),
+		kv:              spkv,
+		safePoint:       0,
+		spTime:          time.Now(),
+		closed:          make(chan struct{}),
+		replicaReadSeed: rand.Uint32(),
```

Comment:
Is it better to move this `seed` to `RegionStore`?

Reply:
We discussed about this last time and agreed to make load balance policy a per session setting. We can have more discussion if it makes better sense to change.

Reply:
In my opinion, if move this `seed` to `RegionStore`, we can make load balance for each store of one region. But it can't make a balance for per session.

Reply:
It was like that and changed to what you see right now. The reason behind it was making follower read target stable per session and simplifying region cache. If it is possible can you talk to @coocood & @overvenus and get more context about why we come up with this last time.

Reply:
Fine.